.gitignore

```diff
@@ -13,6 +13,8 @@ cmake-build-*/
 !.vscode/extensions.json
 *.swp
 *~
+**/_ReSharper*
```

Comment:
I'm not sure how this line got in here. Is this necessary there a need for this, or did you intend to remove the `**/` in 4f9676672e512c9fdff2adaf55b9d2b059bec8e0?

Reply:
Yes I wanted to remove the `**/` in this commit. Well Resharper is a very popular plugin from Jetbrains for VS that gives CLion capabilities to the IDE. This is the folder that Resharper writes its chase files.

Reply:
```suggestion
_ReSharper*
```